### PR TITLE
add intent: quote

### LINKED ISSUE #10: Add new intent: quote
https://github.com/the-vision/jarvis-core/wiki

### PR DESCRIPTION
**Pull request by srishti-nema**
Resolves #10 .

### CODE REVIEW

**Review by RohitRathore1 — APPROVED** [review]

**Comment by srishti-nema** [comment]
Should I update test.py and commit??

**Comment by RohitRathore1** [comment]
@srishti-nema Yes, you have to update.

**Review by RohitRathore1 — APPROVED** [review]

File: data/nlu.md [hunk 1]
```diff
@@ -10,7 +10,14 @@
...
 
+
 ## intent:hello
 - hey
 - hello
 - hi
+
+## intent:quote
+- quote
+- quote of the day
```
Comment (swapagarwal) [inline data/nlu.md]: I'm editing this query as it would require special handling compared to the other two. 😅

**Comment by swapagarwal** [comment]
Thanks @srishti-nema! 👍
Here's a good issue for you to work on next: https://github.com/the-vision/jarvis-telegram/issues/10

**Pull request merged**